api_types.ts

```diff
@@ -704,12 +704,18 @@ export interface Sync {
    * value returned in the `continuation` property of result of the prior sync.
    */
   continuation?: Continuation;
+
+  // TODO(patrick): Unhide this
+  /** @hidden */
+  prevSyncContinuation?: Continuation;
```

Comment:
The name `prevSyncContinuation` doesn't feel that distinct from `continuation`

I wonder if it's worth doing a quick naming brainstorm & voting exercise in a doc? Personally I might lean toward a name more like `incrementalSyncBase` to clearly distinguish it from non-incremental sync.

Reply:
Ah you're right, I'll make a doc.

Reply:
NARRATOR: He did not make a doc